experiments/many_well/ground_truth_higher_order_marginals.py

```diff
@@ -0,0 +1,44 @@
+import numpy as np
```

Comment:
Make a histogram showing the frequency of how often higher order marginals are sampled from. Not used in the paper, but nice to have in the repo if we come back to this problem.